templates/index.html

```diff
@@ -15,9 +15,30 @@
         <div class="container">
             <div class="col-md-6 col-md-offset-3">
                 <div class="page-header">
-                    <h2>REDCap to Harvest</h2>
-
-                    <p class="lead">Preview and interact with a live Harvest application built directly from your REDCap project!</p>
+		    <div class="row">
```

Comment:
Fix indentation of HTML markup. You should be able to scan down the lines and know exactly how nested each element is. Each indent is four spaces.